packages/console/src/components/Markdown/index.module.scss

```diff
@@ -30,6 +30,21 @@
     }
   }
 
+  a {
+    font: var(--font-body-medium);
+    color: var(--color-text-link);
+    text-decoration: none;
+    text-underline-offset: 2px;
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &:active {
+      color: var(--color-primary-pressed);
+    }
+  }
```

Comment:
I'm wondering what broke it? Have you found out the root cause?

Reply:
I removed the original CSS by mistake 😢; I shall update the old CSS.